styles.css

```diff
@@ -0,0 +1,674 @@
+    font-size: 40px;
+    position: absolute;
+    right: 40px;
+    z-index: 9;
+  }
+}
+
+/* Desktop */
+
+@media screen and (min-width: 768px) {
+  .flist {
+    background-color: black;
+    display: flex;
+    justify-content: flex-end;
+    margin: 0;
+    gap: 20px;
+  }
+
+  .event {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 121px;
+    padding-left: 130px;
+  }
+
+  .fa-facebook-f {
+    color: #d3d3d3;
+  }
+
+  .flist a {
+    text-decoration: none;
+    color: #d3d3d3;
+    display: flex;
+    list-style: none;
+  }
+
+  .logo a {
+    display: flex;
+    color: black;
+    text-decoration: none;
+    place-items: center;
+    padding-top: -8px;
+    padding-left: 50px;
+  }
+
+  .bar {
+    display: none;
+  }
+
+  .lnav {
+    background-color: white;
+    height: 50px;
+    padding-top: 5px;
+    padding-left: 10px;
+    padding-right: 10px;
+    justify-content: space-between;
+  }
+
+  .barlist {
+    display: flex;
+    gap: 35px;
+    justify-content: flex-end;
+    list-style: none;
+    margin-top: -54px;
+    line-height: 20px;
+  }
+
+  .barlist li {
+    list-style: none;
+    margin: 14px 10px;
+  }
+
+  .barlist a {
+    text-decoration: none;
+    color: black;
+    display: flex;
+    list-style: none;
+  }
+
+  .barlist a.active {
+    color: var(--secondary-dark);
+  }
+
+  .barlist a:hover {
+    color: #ec5242;
+  }
+
+  .barlist button {
+    border: #ec5242 2px solid;
+    background-color: white;
+    color: #ec5242;
+    font-size: 18px;
+  }
+
+  .barlist button:hover {
+    background-color: #ec5242;
+    color: white;
+  }
+
+  .hello {
+    padding: 10px 40px;
+    display: flex;
+  }
+
+  .hello h1 {
+    font-size: 30px;
+    font-weight: 700;
+    color: #ec5242;
+  }
+
+  .hello h2 {
+    text-transform: uppercase;
+    font-weight: 900;
+    line-height: 55px;
+    font-family: sans-serif;
+    background-image: linear-gradient(rgba(255, 38, 0, 0.76), rgba(255, 51, 0, 0.575)), url("images/lbackground.jpg");
+    color: transparent;
+    -webkit-background-clip: text;
+    font-size: 45px;
+  }
+
+  .hello p {
+    background-color: #f5f5f5;
+    border: #d3d3d3 1px solid;
+    padding: 10px;
+    font-size: 17px;
+    line-height: 30px;
+    width: 550px;
+    font-weight: 500;
+    color: #272a31;
+    font-family: "Lato", sans-serif;
+    margin: 20px;
+  }
+
+  .pro-list {
+    border: #d3d3d3 2px solid;
+    border-radius: 5px;
+    padding: 10px;
+    display: flex;
+    flex-direction: column;
+    background-color: rgba(255, 255, 255, 0.1);
+    gap: 20px;
+    margin: 20px;
+    align-items: center;
+  }
+
+  .pro-list h3 {
+    color: #ec5242;
+    font-weight: bold;
+    text-align: center;
+  }
+
+  .pro-list p {
+    color: #fff;
+    text-align: center;
+    font-size: 15px;
+  }
+
+  .program-list {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .pro-list img {
+    width: 71px;
+    padding: -4px;
+  }
+
+  .persons {
+    width: 100%;
```

Comment:
For the program section on the desktop view:

- [ ] Please add the correct padding-top, padding-left, and padding-right to the whole section.
- [ ] Please make the width of the program cards equal and decrease the gap between them like the given design
- [ ] please fix the styling of the link by removing the orange background and adding only the underline to it and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/170764227-5cc9cc75-2b72-4fec-bb1d-b0810f4addac.png) | ![image](https://user-images.githubusercontent.com/68796232/170764208-95f4dea4-aec6-4f07-b550-f593b1582063.png)  |